From 2e584b10b97fb75b93aff7be217f8a8a294455d4 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Thu, 16 Mar 2017 15:46:10 +0200
Subject: [PATCH 1483/2241] dts: a80x0: add phy 3310 handler for MaciatoBIN
 device tree

- CP0 Port 0 connected to CP0 MDIO XSMI 0
- CP1 Port 0 connected to CP0 MDIO XSMI 8

Change-Id: I001c3e6ef9d8c778f332fc8d133a4552649feeec
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37523
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index e8244c6..86daf03 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -127,11 +127,24 @@
 					status = "okay";
 				};
 			};
+			mdio@15b000 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				phy0: ethernet-phy@0 {
+					compatible = "ethernet-phy-ieee802.3-c45";
+					reg = <0>;
+				};
+				phy1: ethernet-phy@1 {
+					compatible = "ethernet-phy-ieee802.3-c45";
+					reg = <8>;
+				};
+			};
 			gop {
 				emac0_0: mac0 {
 					phy-mode = "sfi"; /* lane-4 */
 					phys = <&comphy0 4 COMPHY_SFI>;
 					phy-names = "comphy";
+					phy = <&phy0>;
 				};
 			};
 			ppv22@000000 {
@@ -202,6 +215,7 @@
 					phy-mode = "sfi"; /* lane-4 */
 					phys = <&comphy1 4 COMPHY_SFI>;
 					phy-names = "comphy";
+					phy = <&phy1>;
 				};
 				emac2_1: mac2 {
 					phy-mode = "sgmii"; /* lane-0 */
-- 
2.7.4

